nb2workflow/nbadapter.py

```diff
@@ -249,14 +250,15 @@ class NotebookAdapter:
     limit_output_attachment_file = None
 
 
-    def __init__(self, notebook_fn, tempdir_cache=None, n_download_max_tries=10, download_retry_sleep=.5):
+    def __init__(self, notebook_fn, tempdir_cache=None, n_download_max_tries=None, download_retry_sleep=None, max_download_size=None):
```

Comment:
The code will be cleaner if you just pass config object to the constructor and get specific parts inside